src/utilities/theme/types.ts

```diff
@@ -45,6 +47,7 @@ export interface ThemeConfig {
     topBar?: Record<string, string>;
   };
   UNSTABLE_colors?: Partial<RoleColors>;
+  mode?: Mode;
```

Comment:
What do you think about making `mode` a prop on `ThemeProvider`? So eventually it is something like:

```tsx
<ThemeProvider theme={{primary: '#FF0000'}} mode="light">
```

Maybe there is a potential to align to naming with color theory and the media query which this is referred to as a color scheme.

- https://en.wikipedia.org/wiki/Color_scheme
- https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme